content/flux/v0.x/spec/options.md

```diff
@@ -50,18 +44,17 @@ option task = {
 }
 ```
 
-##### location
+#### location
 The `location` option sets the default time zone of all times in the script.
 The location maps the UTC offset in use at that location for a given time.
-The default value is set using the time zone of the running process.
+The default value is [`timezone.utc`](/flux/v0.x/stdlib/timezone/#constants).
 
 ```js
-option location = fixedZone(offset:-5h) // Set timezone to be 5 hours west of UTC.
-option location = loadLocation(name:"America/Denver") // Set location to be America/Denver.
-```
+// Set timezone to be 5 hours west of UTC.
+option location = timezone.fixed(offset: -5h)
```

Comment:
Do we need to mention the `import "timezone"`?

Reply:
Oh good call. I'll add it.